Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -259,7 +319,7 @@ std::unique_ptr<CSSParserSelector> CSSSelectorParser::consumeComplexSelector(CSS
     return selector;
 }
 
-std::unique_ptr<CSSParserSelector> CSSSelectorParser::consumeRelativeSelector(CSSParserTokenRange& range)
+std::unique_ptr<CSSParserSelector> CSSSelectorParser::consumeRelativeSelector(CSSParserTokenRange& range, auto& postOperationOnCombinator)
```

Comment:
It seems to me that trying to use the existing `consumeRelativeSelector` just increases complexity here as there is little code left in it to share. It might be better to have separate `consumeRelativeScopeSelector` and `consumeRelativeNestedSelector` functions for now.